docker-compose.yml

```diff
@@ -8,3 +8,7 @@ services:
       - DEEPL_API_KEY=${DEEPL_API_KEY}
       - WHISPER_API_KEY=${WHISPER_API_KEY}
       - OPENAI_API_KEY=${OPENAI_API_KEY}
+      - APP_ENV=${APP_ENV:-'production'}
```

Comment:
The `CMD` modification in the [Dockerfile](#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557) changes the service to the Ruby Web App. We should rename the existing `transcriber` service in docker compose, or keep the existing service and add a new service for it.

Reply:
Opted to replace `transcriber` service with the Sinatra web app completely.